cv19ResSupportV3/Startup.cs

```diff
@@ -35,7 +35,7 @@ public Startup(IConfiguration configuration)
         private const string ApiName = "cv-19-resident-support";
 
         // This method gets called by the runtime. Use this method to add services to the container.
-        public static void ConfigureServices(IServiceCollection services)
+        public void ConfigureServices(IServiceCollection services)
```

Comment:
It uses 'Configuration'  when it sets up the DB Context - which is an instance variable so can no longer be static.

Reply:
I'm fine with the change. It's just confusing - it was working before just fine :confused: , or wasn't it?

It was successfully configuring the DBContext, Gateway, and UseCase instances.
![image](https://user-images.githubusercontent.com/43747286/136205654-a5f7eea1-5330-4e5a-93b5-6f88f0af102f.png)

Even after your change, the Gateway and Usecase instances are still getting configured using static methods.

Reply:
I'd recommend re-adding the static modifier locally to see why it needs to be removed. In summary.. if you look at the top:

It sets 'Configuration' in the constructor:

![image](https://user-images.githubusercontent.com/80220779/136210177-0330d0b0-bef3-405e-8db5-2699c25e6a61.png)

Configuration is an instance variable, in that the class needs to be instantiated to use it. Configuration doesn't and shouldn't have a static modifier.

Then if you go to ConfigureDbContext() I've now used Configuration in what was a static method. Because it's an instance variable ConfigureDbContext can no longer be static in C#.

ConfigureServices() calls ConfigureDbContext(), which means ConfigureServices() can no longer be static.

RegisterGateways and RegisterUseCases are still static. They can be called from non-static or static methods so remain unchanged.

Hopefully that clarifies. If you go into Visual Studio and add 'static' back to where it was removed, you'll see it won't build and will give this reason.